### ISSUE TITLE
pretraining and finetuning documentation: nnUNetv2_preprocess step fails

### ISSUE THREAD

**Issue by snorthman** [MIC-DKFZ/nnUNet#2560]
Following the finetuning steps described [here](https://github.com/MIC-DKFZ/nnUNet/blob/master/documentation/pretraining_and_finetuning.md)...

```shell
PRETRAINING='111'
FINETUNING='112'

nnUNetv2_plan_and_preprocess -pl 'nnUNetPlannerResEncM' -d $FINETUNING --verify_dataset_integrity -c '2d'
nnUNetv2_extract_fingerprint -d $PRETRAINING
nnUNetv2_move_plans_between_datasets -s $FINETUNING -t $PRETRAINING -sp 'nnUNetResEncUNetMPlans' -tp 'nnUNetResEncUNetMPlans2'
nnUNetv2_preprocess -d $PRETRAINING -plans_name 'nnUNetResEncUNetMPlans2' -c '2d'
```

The final step, nnUNetv2_preprocess, gives:

`FileNotFoundError: [Errno 2] No such file or directory: '/nnunet_data/nnUNet_preprocessed/Dataset111_MyDataSet/dataset.json'`

The solution is ofcourse simple, but I do think it is an unexpected error.. either I did something wrong, causing the file not to exist, or `nnUNetv2_move_plans_between_datasets` unintentionally(?) does not copy over `dataset.json`.

EDIT:

And yes, the issue is indeed resolved by putting

```
cp /nnunet_data/nnUNet_preprocessed/Dataset112_MyDataSet/dataset.json /nnunet_data/nnUNet_preprocessed/Dataset111_MyDataSet/dataset.json
```

in between `nnUNetv2_move_plans_between_datasets` and `nnUNetv2_preprocess`, but it's a bit silly.

**Comment by dojoh** [comment]
I believe this is intended as the labels might differ between datasets.

**Comment by Lars-Kraemer** [comment]
I agree with @dojoh that this is intended behavior, since the two datasets do not necessarily have to be identical.
For the simple case where both datasets share the same relevant parameters, the easiest solution is to just copy the dataset.json file. But this is not a general solution.

Best,
Lars
